content/en/data_jobs/databricks.md

```diff
@@ -23,7 +23,9 @@ Follow these steps to enable Data Jobs Monitoring for Databricks.
 ### Configure the Datadog-Databricks integration
 
 1. In your Databricks workspace, go to **Settings** > **Developer**. Next to **Access tokens**, click **Manage**.
-1. Click **Generate new token**, enter a comment, and click **Generate**. Take note of your token.
+1. Click **Generate new token**, enter "Datadog Integration" in the **Comment** field, remove the default value in **Lifetime (days)**, and click **Generate**. Take note of your token. **Important:** Make sure you delete the default value in **Lifetime (days)** so that the token doesn't expire and the integration doesn't break.
+1. As an alternative, follow the [official Databricks documentation][10] to generate access token for a [service principal][11].
+1. Ensure the account generating the token has [CAN VIEW access][9] for the Databricks jobs you want to monitor, and that the token is configured to never expire.
```

Comment:
```suggestion
1. Ensure the account generating the token has [CAN VIEW access][9] for the Databricks jobs and clusters you want to monitor, and that the token is configured to never expire.
```

Reply:
Also, the "as an alternative..." and "ensure account generating token ...." shouldn't be explicity steps, but additional details/alternatives on step 2